feature/onboarding/welcome/src/main/kotlin/app/k9mail/feature/onboarding/welcome/ui/WelcomeContent.kt

```diff
@@ -109,6 +116,23 @@ private fun WelcomeLogo(
     }
 }
 
+
+@Composable
+private fun WelcomeTitleItem(
+    title: String,
```

Comment:
Please add the modifier here:

```suggestion
    title: String,
    modifier: Modifier = Modifier,
```